templates/partial/contact_info.html

```diff
@@ -1,19 +1,21 @@
 <div class="govuk-body">
-  <h2 class="govuk-heading-s govuk-!-margin-bottom-1">Request access</h2>
-  <p id="request_access" class="govuk-body">
+  {% if entity_type != "Publication collection" and entity_type != "Publication dataset" %}
```

Comment:
need to add charts and dashboards here. Wondering if there's a better way to do it though in case there are charts/dashboards added that are not public. Could we infer it via the `security_classification` which mitch has just added, where `Official` maps to the previous `audience: Published`

Reply:
Just seen this also might change, https://github.com/ministryofjustice/find-moj-data/issues/1305